### ISSUE TITLE
Not showing sheet anymore in 1.8

### ISSUE THREAD

**Issue by chgiaimo** [jopeek/fvtt-loot-sheet-npc-pf2e#6]
The sheet can't be selected to e used in this pf2 version

**Comment by brandon-lewis** [comment]
Same problem here - using Foundry version 0.6.4 and Pathfinder Second Edition version 1.7.0.

**Comment by StopSignGaming** [comment]
is this still being worked on?

**Comment by jopeek** [comment]
Yeah sorry. I should be able to fix this tomorrow.

**Comment by jopeek** [comment]
I take it back. I do not personally use Pf2e and it looks like they've refactored a bunch of code, removing some things I needed for the sheet. I'm sure it's possible to fix, but I do not have the time to do it. I'm happy to accept pull requests from anyone that fixes the issue.

**Comment by brandon-lewis** [comment]
@jopeek I have a strong need for this kind of feature based on the looks and might be able to commit some time. Any quick pointers on where to start or where the bug is originating?

**Comment by jopeek** [comment]
Sure, yeah, the very first statement on my .js file is the issue. The actor.js file I'm trying to import no longer exists. There's been some re-factoring of the Pf2e system, but I'm not sure where they've moved it, if it still exists, etc. It might be a good idea to look at how a different sheet is set up, one that still works with the latest Pf2e version.

**Comment by chgiaimo** [comment]
Well the NPC sheet is now in the main.bundle.js
Would this help?

**Comment by brandon-lewis** [comment]
> Well the NPC sheet is now in the main.bundle.js
> Would this help?

Yes, I did find the new location of the class very quickly. The problem for me is I have a very rudimentary understanding of webpack and ESModules, and I don't currently know how to properly import the relocated actor sheet class.

**Comment by rsimoes** [comment]
You can get the sheet class from CONFIG.Actor.entityClass (no module import necessary).
